sphinx_click/ext.py

```diff
@@ -25,6 +25,22 @@
 ANSI_ESC_SEQ_RE = re.compile(r'\x1B\[\d+(;\d+){0,2}m', flags=re.MULTILINE)
 
 
+def _process_lines(event_name):
+    def decorator(func):
```

Comment:
Can we add [`functools.wraps`](https://docs.python.org/3/library/functools.html#functools.wraps) to preserve the docstrings etc.? Also, type annotations?